packages/client/hmi-client/src/components/models/tera-strata-model-diagram.vue

```diff
@@ -119,16 +118,18 @@ const emit = defineEmits([
 	'close-preview',
 	'asset-loaded',
 	'close-current-tab',
-	'update-model-content'
+	'update-model-content',
+	'all-nodes-typed'
 ]);
 
 const props = defineProps<{
 	model: Model;
 	showTypingToolbar: boolean;
 	typeSystem?: TypeSystem;
+	showReflexivesToolbar: boolean;
 }>();
 
-const typedModel = ref<Model>();
+const typedModel = ref<Model>(props.model);
 
 const newModelName = ref('New Model');
```

Comment:
Unrelated, but since you are in the area, it doesn't seem like newModelName is used, can you remove?